### PR TITLE
add linkerd install doc

### PR DESCRIPTION
**Pull request by ilackarms**

### CODE REVIEW

File: supergloo/docs/mesh/install-linkerd.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+# reopened with the relevant failures.
+#
+apiVersion: supergloo.solo.io/v1
+kind: Install
+metadata:
+  name: linkerd
+  namespace: supergloo-system
+spec:
+   ## add the following line 
+   disabled: true
```
Comment (christian-posta) [inline supergloo/docs/mesh/install-linkerd.md]: Could you add highlighting here? It took me a couple takes to figure out what needs to be changed in the YAML

**Comment by christian-posta** [comment]
So now that we have multiple mesh to install... we'll need to clearly identify to which mesh technology each of our tutorials applies. For example.. Traffic Shifting won't work with Linkerd yet.

**Comment by ilackarms** [comment]
yes, my next pr (which will include a linkerd tutorial) will group tutorials by their mesh type. i think for now it doesn't need to be made part of this PR as the existing tutorials require istio as a prereq

File: supergloo/docs/mesh/install-linkerd.md (hunk 1 above)
Comment (ilackarms) [reply supergloo/docs/mesh/install-linkerd.md]: adding now

**Comment by ilackarms** [comment]
@christian-posta i think this PR does that already 
Screenshot from 2019-04-16 17-00-25

**Pull request merged**